src/main/java/com/exadel/discountwebapp/vendor/mapper/VendorMapper.java

```diff
@@ -0,0 +1,56 @@
+package com.exadel.discountwebapp.vendor.mapper;
+
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.location.service.LocationService;
+import com.exadel.discountwebapp.vendor.entity.Vendor;
+import com.exadel.discountwebapp.vendor.vo.VendorRequestVO;
+import com.exadel.discountwebapp.vendor.vo.VendorResponseVO;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.modelmapper.convention.MatchingStrategies;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class VendorMapper {
+    private final LocationService locationService;
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    @Autowired
+    public VendorMapper(LocationService locationService) {
```

Comment:
replace with requiredargsconstructor please
and don't forget to remove final for private final ModelMapper modelMapper = new ModelMapper(); in this case